securedrop/secure_tempfile.py

```diff
@@ -1,12 +1,16 @@
 import base64
 import os
 from tempfile import _TemporaryFileWrapper
+# python-gnupg will not recognize our SecureTemporaryFile as a stream-like type
+# and will attempt to call encode on it, thinking it's a string-like type. To
+# avoid this we add it the list of stream-like types.
+from gnupg._util import _STREAMLIKE_TYPES
+_STREAMLIKE_TYPES.append(_TemporaryFileWrapper)
```

Comment:
I just put it at the top because I liked the formatting better (i.e., having it right next to the import block and then leaving the rest of the file to the sft class), and because `_TemporaryFileWrapper` is also a stream-like type. I doubt it's a type will ever be used in SecureDrop, but at least it's correct appended.

So maybe it could matter less to you to see it moved to the bottom, and instead it's really important to you that we're as granular as possible. Or maybe you were just curious.

Personally, I don't see much of a difference either way, so if you feel passionately about one approach, we can change it.